packages/sanity/src/_internal/manifest/manifestTypes.ts

```diff
@@ -6,15 +6,14 @@ export type ManifestSerializable =
   | ManifestSerializable[]
 
 export interface CreateManifest {
-  version: 1
+  version: number
   createdAt: string
   workspaces: ManifestWorkspaceFile[]
 }
 
-export interface ManifestWorkspaceFile {
-  name: string
-  dataset: string
+export interface ManifestWorkspaceFile extends Omit<CreateWorkspaceManifest, 'schema' | 'tools'> {
   schema: string // filename
+  tools: string // filename
```

Comment:
what does the filename comment refer to here?

Reply:
The string is a filename